esda/moran.py

```diff
@@ -1785,6 +1784,7 @@ def _explore_local_moran(moran_local, gdf, crit_value, **kwargs):
     m
         folium.Map
     """
+    from matplotlib import colors
```

Comment:
Could you nest this LOC with try-except statement to assure that user has matplotlib installed in their environment, as done in [`topo`](https://github.com/pysal/esda/blob/a337fc42dbdb315181f6b93ee3935e76fb84aebe/esda/topo.py#L121)?